app/uk/gov/hmrc/exports/services/WcoSubmissionService.scala

```diff
@@ -41,6 +44,8 @@ class WcoSubmissionService @Inject()(
       .getOrElse(throw new IllegalArgumentException("An DUCR is required"))
     val payload = wcoMapperService.toXml(metaData)
 
+   logger.debug(s"Sending $payload for $declaration")
```

Comment:
Is this allowed/advisable?   Beth suggested it would be useful to see what xml we are sending.

Reply:
I would not do that, because this will contain PII and then we will require to clean up the logs when we need it in production.
You can create a `printNonPII` method that prints what you all the payload without PII "EORI, ADDRESSES, NAMES, etc"

Reply:
First we need to understand what is PII, which I believe a EORI is one, but Ian is asking that

Reply:
I wondered about that.  Beth said we dump the dec json at the frontend.   I'll get rid of logging then.

Reply:
we will have to stop dumping the payload... unless we filter it

Reply:
I've removed the logging.   If its needed we can create a ticket.   Was just a (poor) idea of mine

Reply:
On level of debug should be fine. This is exactly like logging guide suggest